CMakeLists.txt

```diff
@@ -106,9 +106,6 @@ add_subdirectory(config)
 # this is where the plugins to 3rd party software live
 add_subdirectory(plugins)
 
-# this is where configuration for example data sets lives
-add_subdirectory(examples)
```

Comment:
I think we should leave this in along with the CMakeLists.txt file in the examples directory.  This just installs whatever is in the examples to include it with the installer.  For now that would just be the EXAMPLES.rst, but that's okay.  We may add more later.